evap/staff/tests/test_views.py

```diff
@@ -7,8 +8,8 @@
 from model_mommy import mommy
 import xlrd
 
-from evap.evaluation.models import Semester, UserProfile, Course, CourseType, \
-                                   TextAnswer, Contribution
+from evap.evaluation.models import Semester, UserProfile, Course, CourseType, TextAnswer, Contribution, Questionnaire,\
+    Question
```

Comment:
I have not really put any thought into it, so we can do it either way. I found this article but the solutions there are even uglier, even if technically correct. http://stackoverflow.com/questions/17273847/long-imports-in-python